pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Namespaces.java

```diff
@@ -3263,6 +3263,20 @@ void setIsAllowAutoUpdateSchema(String namespace, boolean isAllowAutoUpdateSchem
      */
     void setOffloadPolicies(String namespace, OffloadPolicies offloadPolicies) throws PulsarAdminException;
 
+    /**
+     * Remove the offload configuration for all the topics in a namespace.
```

Comment:
Maybe this is not exactly. Removing the namespace offload policies not represents to remove the topic offload policies, if the topic has its own offload policies, it will not be affected.